src/VirtoCommerce.Platform.Data/Extensions/ApplicationBuilderExtensions.cs

```diff
@@ -12,8 +13,12 @@ public static class ApplicationBuilderExtensions
     {
         public static IApplicationBuilder UseDbTriggers(this IApplicationBuilder appBuilder)
         {
+            var lastChangesService = appBuilder.ApplicationServices.GetRequiredService<ILastModifiedDateTime>();
+
             Triggers<IAuditable>.Inserting += entry =>
             {
+                lastChangesService.Reset(entry.Entity.GetType().FullName);
```

Comment:
Need to consider that DB entities can be inherited by other types. You need to use the original type name of the base type instead of the actual.